tests/epyccel/test_epyccel_lists.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring
```

Comment:
This change is no longer needed. There are no classes in this file

Reply:
fixed in b9d140fcf30aa56f70f2a360875aa18856b97bc5